Lesson3/2.js

```diff
@@ -0,0 +1,29 @@
+/*
+2. Кэширование
+
+Замыкания можно использовать для сохранения состояния и дальнейшей работы с ним.
+
+Реализуйте функцию cache, которая вернет функцию, возводящую число в степень и возвращающую результат. 
+Функция должна запоминать аргументы, которые она уже получала и возвращать результат сразу, не вычисляя его повторно
+
+Пример:  
+const calculate = cache();
+
+calculate(3, 3); // { value: 27, fromCache: false}
+calculate(2, 10); // { value: 1024, fromCache: false}
+calculate(2, 10); // { value: 1024, fromCache: true}
+*/
+
+function cache(x, y){
```

Comment:
Кажется, тут аргументы лишние
```suggestion
function cache(){
```